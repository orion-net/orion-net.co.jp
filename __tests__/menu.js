import React from "react"
import renderer from "react-test-renderer"

import Main from "../src/pages/menu"

describe("menu", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Main />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
