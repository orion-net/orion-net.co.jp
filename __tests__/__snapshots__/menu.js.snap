// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`menu renders correctly 1`] = `
Array [
  <div
    className="hamburger-menu"
  >
    <div
      className="bar-wrapper closed"
      onClick={[Function]}
    >
      <div
        className="bar bar0"
      />
      <div
        className="bar bar1"
      />
      <div
        className="bar bar2"
      />
    </div>
    <ul
      className="navbar closed"
      onClick={[Function]}
    >
      <li
        className=""
      >
        <a
          aria-current="page"
          className=""
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          style={Object {}}
        >
          トップページ
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/concept"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          サロン紹介
        </a>
      </li>
      <li
        className="current_page_item"
      >
        <a
          href="/menu"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          メニュー
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/custom"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          商品紹介
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/custom2"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          おすすめ
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/custom1"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          ご予約
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/access"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          アクセス
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/concept1"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          成人式・七五三
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/contact"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          お問い合わせ
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/privacy"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          プライバシーポリシー
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/custom3"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          成人式・スナップ集
        </a>
      </li>
    </ul>
  </div>,
  <div
    className="layout"
  >
    <div
      onClick={[Function]}
    >
      <div
        className="logo"
      >
        <a
          aria-current="page"
          className=""
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          style={Object {}}
        >
          <img
            alt="オリオン美容室のトップ"
            height="353px"
            src="/img/top3.png"
            width="707px"
          />
        </a>
      </div>
      <div
        className="article-wrapper"
      >
        <h1>
          メニュー
        </h1>
        <div
          className="menu"
        >
          <h2>
            カット
          </h2>
          <dl>
            <dt>
              カット＆ブロー
            </dt>
            <dd>
              ４,４００　（税込）
            </dd>
            <dt>
              中学生
            </dt>
            <dd>
              ３，８５０　（税込）
            </dd>
            <dt>
              小学生
            </dt>
            <dd>
              ２，７５０　（税込）
            </dd>
            <dt>
              小学生未満
            </dt>
            <dd>
              １，６５０　（税込）
            </dd>
            <dt>
              前髪・眉カット
            </dt>
            <dd>
              各 ５５０　（税込）
            </dd>
          </dl>
          <h2>
            パーマ (カット別）
          </h2>
          <dl>
            <dt>
              パーマ
            </dt>
            <dd>
              ３，８５０　（税込）より
            </dd>
            <dt>
              ホット系パーマ
            </dt>
            <dd>
              ８，８００　（税込）より
            </dd>
            <dt>
              ストレートパーマ
            </dt>
            <dd>
              ９，３５０　（税込）より
            </dd>
            <dt>
              ノーアイロン　ストレート
            </dt>
            <dd>
              ５、５００　（税込）より
            </dd>
            <dt>
              ポイントパーマ
            </dt>
            <dd>
              ３，８５０　（税込）より
            </dd>
          </dl>
          <h2>
            カラー　(カット別）
          </h2>
          <dl>
            <dt>
              リタッチカラー
            </dt>
            <dd>
              ４，４００　（税込）より
            </dd>
            <dt>
              バージンカラー
            </dt>
            <dd>
              ４，９５０　（税込）より
            </dd>
            <dt>
              ポイントカラー
            </dt>
            <dd>
              ２，７５０　（税込）より
            </dd>
          </dl>
          <h2>
            マニキュア　(カット別）
          </h2>
          <dl>
            <dt>
              リタッチ
            </dt>
            <dd>
              ４，４００　（税込）より
            </dd>
            <dt>
              バージン
            </dt>
            <dd>
              ４，９５０　（税込）より
            </dd>
          </dl>
          <h2>
            トリートメント
          </h2>
          <dl>
            <dt>
              トリートメント
            </dt>
            <dd>
              １，８７０　（税込）より
            </dd>
            <dt>
              水トリートメント　（スチーム）
            </dt>
            <dd>
              ５５０　（税込）より
            </dd>
          </dl>
          <h2>
            シャンプー・セット
          </h2>
          <dl>
            <dt>
              シャンプー
            </dt>
            <dd>
              １，１００　（税込）より 　※髪の長さにより異なります
            </dd>
            <dt>
              ワンシャンプー
            </dt>
            <dd>
              ５５０　（税込）より
            </dd>
            <dt>
              セット
            </dt>
            <dd>
              １，８７０　（税込）より　※髪の長さにより異なります
            </dd>
            <dt>
              アップ
            </dt>
            <dd>
              ３，３００　（税込）より ※ スタイルによって異なります
            </dd>
          </dl>
          <h2>
            エステ(フェイシャル）・メイク
          </h2>
          <dl>
            <dt>
              エステ
            </dt>
            <dd>
              ３，３００　（税込）
            </dd>
            <dt>
              エステ（フル）
            </dt>
            <dd>
              ５、５００　（税込）
            </dd>
            <dt>
              炭酸パック
            </dt>
            <dd>
              ５５０　（税込）
            </dd>
            <dt>
              エステプラセンタ
            </dt>
            <dd>
              ２，２００　（税込）
            </dd>
            <dt>
              水エステ
            </dt>
            <dd>
              ３，８５０　（税込）
            </dd>
            <dt>
              ハンドマッサージ
            </dt>
            <dd>
              ５５０　（税込）
            </dd>
            <dt>
              メイク
            </dt>
            <dd>
              ３，３００　（税込） より
            </dd>
          </dl>
          <h2>
            着付け
          </h2>
          <dl>
            <dt>
              留袖・訪問着
            </dt>
            <dd>
              ８，２５０　（税込）
            </dd>
            <dt>
              振袖
            </dt>
            <dd>
              １１,０００　（税込）
            </dd>
            <dt>
              袴
            </dt>
            <dd>
              ８，８００　（税込）
            </dd>
            <dt>
              浴衣
            </dt>
            <dd>
              ４，４００　（税込）
            </dd>
          </dl>
          <p>
            ※成人式、着付け、ヘアメイクのご予約を承ります。 ※七五三、卒業式等の着付けやヘアーセットもお気軽にお問い合わせください。
          </p>
        </div>
      </div>
      <div
        className="sidebar"
      >
        <div>
          <a
            className="blog-button"
            href="https://ameblo.jp/teamorion/?frm_id=v.mypage-ameblo--myblog--blog"
          >
            ブログ
          </a>
        </div>
        <div>
          <a
            className="facebook-button"
            href="https://www.facebook.com/orionannex.orion"
          >
            Facebook
          </a>
        </div>
        <div
          className="sp-part-top sp-sns-button"
          id="sp-sns-button-1"
        >
          <ul>
            <li>
              <iframe
                allowtransparency="true"
                frameBorder="0"
                scrolling="no"
                src="https://www.facebook.com/plugins/like.php?href=http%3A%2F%2Forion-net.co.jp%2Fprivacy.html&layout=button_count&width=120&height=21&send=true&show_faces=true&share=false&action=like&colorscheme=light&font"
                style={
                  Object {
                    "border": "none",
                    "height": "21px",
                    "overflow": "hidden",
                    "width": "120px",
                  }
                }
              />
            </li>
            <li>
              <a
                className="twitter-share-button"
                data-count="horizontal"
                data-lang="ja"
                href="https://twitter.com/share"
              >
                Tweet
              </a>
              <script
                charSet="utf-8"
                src="https://platform.twitter.com/widgets.js"
                type="text/javascript"
              />
            </li>
            <li>
              <a
                href="https://line.me/R/msg/text/?%0D%0Ahttp%3A%2F%2Forion-net.co.jp%2Fprivacy.html"
              >
                <img
                  alt=""
                  height="20"
                  src="/img/linebutton_horizontal.png"
                  style={
                    Object {
                      "border": "none",
                    }
                  }
                  width="82"
                />
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>,
  <footer
    className="footer"
  >
    <div
      className="footer-link"
    >
      <a
        className="privacy-policy-link"
        href="/privacy"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        プライバシーポリシー
      </a>
    </div>
    <p
      className="copyright"
    >
      Copyright © オリオン美容室, All rights reserved.
    </p>
  </footer>,
]
`;
